app/helpers/api/bapi/getCatalogueItems.js

```diff
@@ -28,6 +29,7 @@ export const getCatalogueItems = async ({ supplierId, catalogueItemType }) => {
 
   logger.info(`Retrieving ${catalogueItemType} catalogue items from BAPI for supplier ${supplierId}`);
   const catalogueItems = await getData({ endpoint, logger });
+  sortItems(catalogueItems, 'name');
```

Comment:
I would have thought here that it would return a new array that is sorted. But the test prove that it works as is 🤷

Reply:
yeah it's cos js uses the same memory positions
was having a bit of test issue because of this earlier overwriting what the expected result should be

Reply:
oh I don't like that. Can you take a copy of the array that is passed to sortItems and return the copied array that is sorted.

Reply:
leave this as is. Deep cloning the array in sortItems could cause further issues.